examples/navier-stokes/navierstokes.c

```diff
@@ -524,11 +524,11 @@ PetscErrorCode SetUpDM(DM dm, problemData *problem, const char *prefix,
       ierr = DMAddBoundary(dm, DM_BC_ESSENTIAL, "slipx", "Face Sets", 0,
```

Comment:
Benign, but the above is like honey spilled under the cabinets for bugs.

Reply:
Lol, thanks for shedding that image now that this neat-freak is even more concerned about hygiene than usual. I'm going to check under my cabinets now.